### DISCUSSION TITLE
Dart Saas deprication warning after upgrading NextJs to 15.0.1

### DISCUSSION THREAD

**Discussion by chrishj59** [primefaces/community#2912] [PrimeReact]
After upgrading NextJs to 15.0.1  I get the following warning messages:
Deprecation The legacy JS API is deprecated and will be removed in Dart Sass 2.0.0
Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.
Source of the warning: styles/layout/layout.scss:3:8:

What needs to be done to resolve these warnings

**Comment by melloware** [comment]
yep PrimeTek will have to update their SASS as its now `use` syntax not `import` but its just a warning right now.
